buildSrc/src/main/kotlin/com/toasttab/protokt/gradle/ProtobufBuild.kt

```diff
@@ -90,17 +85,10 @@ private fun extraClasspath(project: Project, task: GenerateProtoTask): String {
 }
 
 private fun configureSources(project: Project, generatedSourcesPath: String) {
-    val protoktDir = "$generatedSourcesPath/main/protokt"
-
-    project.the<SourceSetContainer>()["main"]
```

Comment:
We don't actually have to do this - the protobuf-gradle-plugin does this automatically. (Test source sets would have been broken if not, but they work.)